chapter_linear-networks/softmax-regression-gluon_vn.md

```diff
@@ -133,14 +133,15 @@ loss = gluon.loss.SoftmaxCrossEntropyLoss()
 ## Optimization Algorithm
 -->
 
-## *dịch tiêu đề phía trên*
+## Thuật toán tối ưu
 
 <!--
 Here, we use minibatch stochastic gradient descent with a learning rate of $0.1$ as the optimization algorithm.
 Note that this is the same as we applied in the linear regression example and it illustrates the general applicability of the optimizers.
 -->
 
-*dịch đoạn phía trên*
+Ở đây, chúng ta sử dụng thuật toán tối ưu hạ gradient ngẫu nhiên theo minibatch với tốc độ học bằng $0.1$.
+Lưu ý rằng việc này giống với khi áp dụng cho bài toán hồi quy tuyến tính, nó chứng minh khả năng ứng dụng tổng quát của phương pháp tối ưu này.
```

Comment:
```suggestion
Lưu ý rằng đây cũng là thuật toán áp dụng cho bài toán hồi quy tuyến tính, và điều này chứng minh khả năng ứng dụng tổng quát của các phương pháp tối ưu.
```

Reply:
```suggestion
Lưu ý rằng cách làm này y chang cách làm ở ví dụ về hồi quy tuyến tính, minh chứng cho tính tổng quát của phương pháp tối ưu hạ gradient ngẫu nhiên.
```

Reply:
@tanduong nguyên văn là `optimizers`, ý tác giả nói các thuật toán tối ưu đều áp dụng được cho các mô hình khác nhau

Reply:
`optimizers` ở đây chỉ các `bộ tối ưu` trong các bài toán khác nhau, nhưng đều dùng SGD ấy. Mình thấy gợi ý của @tanduong đúng hơn.

Reply:
Đúng rồi `optimizer` là `bộ tối ưu` chứ không phải là `phương pháp tối ưu`.

Reply:
@tanduong từ `y chang` nghe không được "academic writing" cho lắm, `giống hệt` quen thuộc hơn.

Reply:
Mình đã sửa suggestion theo 2 nhận xét trên.